docs/sources/mimir/release-notes/v2.7.md

```diff
@@ -0,0 +1,77 @@
+- **Experimental query sharding improvements**:
+  A new configuration parameter, `-query-frontend.query-sharding-target-series-per-shard`, allows query sharding to take into account cardinality of similar requests executed previously when computing the maximum number of shards to use.
+- **Experimental support for native histogram ingestion**:
+  Native histograms can now be ingested. The new per-tenant limit `-ingester.native-histograms-ingestion-enabled` controls whether native histograms are stored or ignored.
+
+### Alertmanager improvements
+
+- **New metrics** The following upstream metrics are now exposed:
+  - `cortex_alertmanager_dispatcher_aggregation_groups`
+  - `cortex_alertmanager_dispatcher_alert_processing_duration_seconds`
+
+### Helm chart improvements
+
+The Grafana Mimir and Grafana Enterprise Metrics Helm chart is now released independently. See the [Grafana Mimir Helm chart documentation](/docs/helm-charts/mimir-distributed/latest/).
+
+## Important changes
+
+In Grafana Mimir 2.7, the default vaules of the following configuration options have changed:
+
+- `-blocks-storage.bucket-store.batch-series-size` is now enabled by default with a value of `5000`.
+- `-blocks-storage.tsdb.retention-period` has changed from `24h` to `13h`.
+- `-ruler.evaluation-delay-duration` has changed from `0` to `1m`.
+
+In Grafana Mimir 2.7, the following configuration option is now deprecated:
+
+- `-blocks-storage.bucket-store.chunks-cache.subrange-size` since there's no benefit to changing the default of `16000`
+
+In Grafana Mimir 2.7, the following options, metrics, and labels have been removed:
+
+- Experimental support for ephemeral storage introduced in Mimir 2.6.0 has been removed.
+  - Following options are no longer available:
+    - `-blocks-storage.ephemeral-tsdb.*`
+    - `-distributor.ephemeral-series-enabled`
+    - `-distributor.ephemeral-series-matchers`
+    - `-ingester.max-ephemeral-series-per-user`
+    - `-ingester.instance-limits.max-ephemeral-series`
+  - The following metrics have been removed:
+    - `cortex_ingester_ephemeral_series`
+    - `cortex_ingester_ephemeral_series_created_total`
+    - `cortex_ingester_ephemeral_series_removed_total`
+    - `cortex_ingester_ingested_ephemeral_samples_total`
+    - `cortex_ingester_ingested_ephemeral_samples_failures_total`
+    - `cortex_ingester_memory_ephemeral_users`
+    - `cortex_ingester_queries_ephemeral_total`
+    - `cortex_ingester_queried_ephemeral_samples`
+    - `cortex_ingester_queried_ephemeral_series`
+  - Additionally, querying using the `{__mimir_storage__="ephemeral"}` selector no longer works. All label values with the `ephemeral-` prefix within the `reason` label of the `cortex_discarded_samples_total` metric are no longer available.
+- The store-gateway default index header reader no longer uses mmap and the mmap-based index header reader has been removed. The following flags have been changed:
```

Comment:
I think this should be mentioned among features. In 2.6 the mmap-based index-header reader was still the default, so in this release we're effectively introducing the mmap-less index-header reader for whoever use the default config.